native/ex_tokenizers/src/trainers.rs

```diff
@@ -137,32 +137,27 @@ fn populate_bpe_options_to_builder(
 ) -> Result<BpeTrainerBuilder, ExTokenizersError> {
     options
         .iter()
-        .fold(Ok(builder), |builder, option| match option {
-            BPEOption::VocabSize(size) => builder.map(|builder| builder.vocab_size(*size)),
-            BPEOption::MinFrequency(frequency) => {
-                builder.map(|builder| builder.min_frequency(*frequency))
-            }
-            BPEOption::SpecialTokens(tokens) => builder
-                .map(|builder| builder.special_tokens(tokens.iter().map(|s| s.into()).collect())),
-            BPEOption::LimitAlphabet(limit) => {
-                builder.map(|builder| builder.limit_alphabet(*limit))
+        .try_fold(builder, |builder, option| match option {
```

Comment:
love this, didn't know about `try_fold`

Reply:
Neither did I until Clippy yelled at me 😂